src/microbe_stage/editor/upgrades/ChemoreceptorUpgradeGUI.tscn

```diff
@@ -10,21 +10,45 @@ anchor_bottom = 1.0
 size_flags_horizontal = 3
 theme = ExtResource( 1 )
 script = ExtResource( 3 )
+TargetTypesPath = NodePath("TargetType")
 CompoundsPath = NodePath("Compound")
+CompoundsLabelPath = NodePath("CompoundLabel")
+SpeciesPath = NodePath("Species")
+SpeciesLabelPath = NodePath("SpeciesLabel")
 MaximumDistancePath = NodePath("Distance")
 MinimumAmountPath = NodePath("Amount")
 ColourPath = NodePath("Colour")
 
 [node name="Label" type="Label" parent="."]
 margin_right = 1280.0
 margin_bottom = 25.0
+text = "TARGET_TYPE_COLON"
+
+[node name="TargetType" type="OptionButton" parent="."]
```

Comment:
As the options are static, they should be defined here in the scene instead of the .cs file. There's a special button in the Godot editor to edit the contents of an OptionButton.